acceptance/lib/helper.rb

```diff
@@ -193,6 +193,7 @@ def puppet_apply_as_puppet_user
     # installed.
     manifest_path = master.tmpfile("puppetserver_manifest.pp")
     herp_path = master.tmpfile("herp")
+    bin_dir = "/opt/puppetlabs/agent/bin/"
```

Comment:
This should be available in the options['puppetbindir'] according to https://github.com/puppetlabs/puppet-server/pull/378/files#diff-3c9fc7388a83ed36154a42ca88d77a1cR12, shouldn't it?